src/api/Shrooms.Domain.ServiceValidators/Validators/FilterPresets/IFilterPresetValidator.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Shrooms.Contracts.DataTransferObjects.FilterPresets;
+using Shrooms.Contracts.Enums;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Shrooms.Domain.ServiceValidators.Validators.FilterPresets
+{
+    public interface IFilterPresetValidator
+    {
+        void CheckIfFilterTypesAreValid(FilterType[] filterTypes);
+
+        void CheckIfFilterTypesContainsDuplicates(FilterType[] filterTypes);
```

Comment:
Types (plural) -> so it should be "ContainDuplicates" (not Contains).

```suggestion
        void CheckIfFilterTypesContainDuplicates(FilterType[] filterTypes);
```